docs/machine-learning/how-to-guides/serve-model-serverless-azure-functions-ml-net.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Deploy ML.NET Model to Azure Functions
+description: Serve ML.NET sentiment analysis machine learning model for prediction over the internet using Azure Functions
+ms.date: 03/08/2019
+ms.custom: mvc,how-to
+#Customer intent: As a developer, I want to use my ML.NET Machine Learning model to make predictions through the internet using Azure Functions
+---
+
+# How-To: Use ML.NET Model in Azure Functions
+
+This how-to shows how individual predictions can be made using a pre-built ML.NET machine learning model through the internet in a serverless environment such as Azure Functions.
+
+## Prerequisites
+
+- [Visual Studio 2017 15.6 or later](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2017) with the ".NET Core cross-platform development" workload and "Azure development" installed. 
+- [Azure Functions Tools](https://docs.microsoft.com/en-us/azure/azure-functions/functions-develop-vs#check-your-tools-version)
+- Pre-trained model. 
+    - Build Your Own [sentiment analysis machine learning model]()
```

Comment:
Maybe something like "Use the [ML.NET Sentiment Analysis tutorial] (../tutorials/sentiment-analysis.md) to build your own model."